coremltools/converters/mil/input_types.py

```diff
@@ -430,35 +430,36 @@ def __init__(self, shapes, default=None):
 
         Parameters
         ----------
-        shapes: list of Shape objects, or Shape-compatible lists.
-            The valid shapes of the inputs.
-
-            If input provided is not a Shape object, but can be converted to a Shape,
-            the Shape object would be stored in ``shapes`` instead.
+        shapes: list of Shape objects, or Shape-compatible lists
+            * The valid shapes of the inputs.
+            * If input provided is not a :py:class:`Shape` object,
+              but can be converted to a :py:class:`Shape`,
+              the :py:class:`Shape` object would be stored in ``shapes`` instead.
+			
 
         default: tuple of int or None
-            The default shape that is used for initiating the model, and set in
-            the metadata of the model file.
-
-            If None, then the first element in ``shapes`` is used.
+            * The default shape that is used for initiating the model, and set in
+              the metadata of the model file.
+            * If ``None``, then the first element in ``shapes`` is used.
+			
 
         Examples
         --------
         .. sourcecode:: python
 
-        sample_shape = ct.EnumeratedShapes(
-            shapes=[
-                (2, 4, 64, 64),
-                (2, 4, 48, 48),
-                (2, 4, 32, 32)
-            ],
-            default=(2, 4, 64, 64)
-        )
-
-        my_core_ml_model = ct.convert(
-            my_model,
-            inputs=[ct.TensorType(name="sample", shape=sample_shape)],
-        )
+			sample_shape = ct.EnumeratedShapes(
```

Comment:
It looks like you're using actual tabs characters here. We should never use tab characters in the coremltools repository. Instead of using a tab character, we should use four spaces.

Reply:
Fixed, took 2 commits because I didn't see one until I turned on viewing hidden characters in my editor.